triggers/service/src/test/scala/com/digitalasset/daml/lf/engine/trigger/ServiceTest.scala

```diff
@@ -191,19 +191,29 @@ class ServiceTest extends AsyncFlatSpec with Eventually with Matchers with Postg
   }
 
   it should "start a trigger after uploading it" in withHttpService(None) { (uri: Uri, client) =>
+    def chk() = {
+      for {
+        resp <- listTriggers(uri, "Alice")
+        result <- parseTriggerIds(resp)
+      } yield result
+    }
+
     for {
       resp <- uploadDar(uri, darPath)
       JsObject(fields) <- parseResult(resp)
       Some(JsString(mainPackageId)) = fields.get("mainPackageId")
       _ <- mainPackageId should not be empty
-
       resp <- startTrigger(uri, s"$testPkgId:TestTrigger:trigger", "Alice")
       triggerId <- parseTriggerId(resp)
-
-      resp <- listTriggers(uri, "Alice")
-      result <- parseTriggerIds(resp)
-      _ <- result should equal(Vector(triggerId))
-
+      _ <- Future {
+        eventually {
```

Comment:
I understand we want to wait until the new trigger has actually started so that the running trigger map and hence `listTriggers` returns the new trigger id. Does `eventually` add a delay so this is the case?

Reply:
yep